tests/trilinos/precondition_muelu_q_iso_q1.cc

```diff
@@ -285,8 +287,10 @@ void Step4<dim>::solve ()
     SolverCG<>              solver (solver_control);
     TrilinosWrappers::PreconditionAMGMueLu preconditioner;
     preconditioner.initialize(preconditioner_matrix);
-    solver.solve (system_matrix, solution, system_rhs,
-                  preconditioner);
+    check_solver_within_range(
+      solver.solve (system_matrix, solution, system_rhs,
+		    preconditioner),
+      solver_control.last_step(), 30, 56);
```

Comment:
I am not happy with the large range, but on my machine with Trilinos 12.0.1 I need 52 and 55 iterations respectively. This is with gcc and with intel 15 (http://cdash.kyomu.43-1.org/testDetails.php?test=17132073&build=10268). The original output had 34 iterations. Just leave it as is?